.github/labeler.yml

```diff
@@ -1,58 +1,31 @@
 ep:DML:
-  - '(DirectML)'
-# Add/remove 'ep:MIGraphX' label if issue contains the word 'MIGraphX'
-ep:MIGraphX:
-  - '(MIGraphX)'
-# Add/remove 'ep:oneDNN' label if issue contains the word 'oneDNN'
-ep:oneDNN:
-  - '(oneDNN)'
-# Add/remove 'ep:OpenVINO' label if issue contains the word 'OpenVINO'
-ep:OpenVINO:
-  - '(OpenVINO)'
-# Add/remove 'ep:RockchipNPU' label if issue contains the word 'Rockchip NPU'
-ep:RockchipNPU:
-  - '(Rockchip NPU)'
-# Add/remove 'ep:TensorRT' label if issue contains the word 'TensorRT'
+  - '/\bdirectml\b/i'
+  - '/\bdml\b/i'
+ep:MIGraphX: /\bmigraphx\b/i'
+ep:oneDNN: '/\bonednn\b/i'
+ep:OpenVINO: '/\bopen\s*vino\b/i'
+ep:RockchipNPU: '/\brockchip\b/i'
 ep:TensorRT:
-  - '(TensorRT)'
-# Add/remove 'ep:tvm' label if issue contains the word 'TVM'
-ep:tvm:
-  - '(TVM)'
-# Add/remove 'ep:VitisAI' label if issue contains the word 'Vitis AI'
-ep:VitisAI:
-  - '(Vitis AI)'
-# Add/remove 'api:JavaScript' label if issue contains the word 'JavaScript'
-api:JavaScript:
-  - '(JavaScript)'
-# Add/remove 'api:Java' label if issue contains the word 'Java' not followed by 'Script'
-api:Java:
-  - '(Java(?!Script))'
-# Add/remove 'api' label if issue contains the word 'C#'
-api:CSharp:
-  - '(C#)'
-# Add/remove 'api' label if issue contains the word 'Python', 'C++', 'C\b', 'Obj-C', or 'WinRT'
-api:
-  - '(Python|C\+\+|C\b|Obj-C)'
-# Add/remove 'platform:jetson' label if issue contains the word 'Jetson' or 'jetson'
-platform:jetson:
-  - '(Jetson|jetson)'
-# Add/remove 'platform:windows' label if issue contains the word 'Windows'
+  - '/\btensor\s*rt\b/i'
+  - '/\btrt\b/i'
+ep:tvm: '/\btvm\b/i'
+ep:VitisAI: '/\bvitus(?:ai)?\b/i'
+platform:jetson: '/\bjetson\b/i'
+platform:mobile:
+  - '/\bobj-?c\b/i'
+  - '/\bnnnapi\b/i'
+  - '/\bcore-?ml\b/i'
 platform:windows:
-  - '(Windows|WinRT)'
-# Add/remove 'model:transformer' label if issue contains the word 'BERT', 'Bert\b', 'bert\b', 'GPT2', 'gpt2', 'Hugging Face', 'HuggingFace', 'Hugging face', 'Huggingface', 'hugging face', 'huggingface', 'Longformer', 'Long former', 'longformer', 'long former', 'T5', or 't5'
+  - '/\bwindows\b/i'
+  - '/\bwinrt\b/i'
 model:transformer:
-  - '(BERT|Bert\b|bert\b|GPT2|gpt2|Hugging Face|HuggingFace|Hugging face|Huggingface|hugging face|huggingface|Longformer|Long former|longformer|long former|T5|t5)'
-# Add/remove 'quantization' label contains the words 'Is this a quantized model\?\n\nYes', 'Quantization', 'quantization', 'Quantized', or 'quantized' not preceded by 'Is this a '
+  - '/\bbert\b/i'
+  - '/\bgpt-?2\b/i'
+  - '/\bhugging-?face\b/i'
+  - '/\blong-?former\b/i'
+  - '/\bt5\b/i'
 quantization:
-  - '(Is this a quantized model\?\n\nYes|Quantization|quantization|Quantized|(?<!(Is this a ))quantized)'
-# Add/remove 'build' label if issue contains the word 'ACL', 'ArmNN', 'CUDA', 'DirectML', 'MIGraphX', 'oneDNN', 'OpenVINO', 'Rockchip NPU', 'TensorRT', 'TVM', 'Vitis AI', 'Java', 'Python', 'C++', 'C#', 'C\b', or 'Obj-C'
+  - '/\bquantiz(?:e|ation)\b/i'
```

Comment:
does this regex include "quantized"? The original query had the full line because the templates have this question. It will overtrigger on every issue with the template if it matches on "quantized"

Reply:
Oh, I see. The original regex also has quantized. So do we only want to match on the question? With a yes answer

Reply:
The question is only in one (?) of the templates, so it might be good to match on "quantization" in addition

Reply:
Oh ok but not quantized?

Reply:
Not "quantized," otherwise it would match on that word for every issue filed using the "performance" template that includes that question.